tests/framework/utils/k8s_helper.go

```diff
@@ -347,7 +347,12 @@ func (k8sh *K8sHelper) ResourceOperationFromTemplate(action string, podDefinitio
 // ResourceOperation performs a kubectl action on a pod definition
 func (k8sh *K8sHelper) ResourceOperation(action string, manifest string) error {
 	args := []string{action, "-f", "-"}
-	logger.Infof("kubectl %s manifest:\n%s", action, manifest)
+	maxManifestCharsToPrint := 2000
+	if len(manifest) > maxManifestCharsToPrint {
+		logger.Infof("kubectl %s manifest (truncated):\n%s", action, manifest[0:maxManifestCharsToPrint])
```

Comment:
Do we need to print the manifest at all?

Reply:
It is nice to see the manifests being applied. This change was made because the crds.yaml was so long, but I'm not actually sure if this is used to print other manifests. If it's only for crds.yaml, agreed it's not needed and i'll skip printing.